apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -283,6 +283,7 @@ class LevelDetailsDialog extends Component {
           />
           <Button
             href={level.url || scriptLevel.url}
+            target={'_blank'}
```

Comment:
lol, I added the logic myself to set `rel` appropriately based on this, and I still felt compelled to go check that it was actually working as expected. That indicates to me that we should just get rid of the `target` property on this component entirely and have something like a `openInNewTab` property that sets both target and rel internally.

That being said, not a blocker for this PR

Reply:
I did actually look for this and found the logic that set `rel`. I was expecting `openInNewTab` as a prop (or something similar) like we have for `SafeMarkdown` so I think that would be a good improvement